config/config.go

```diff
@@ -148,6 +148,8 @@ type Config struct {
 	SkipRegisterToDashboard bool `toml:"skip-register-to-dashboard" json:"skip-register-to-dashboard"`
 	// EnableTelemetry enables the usage data report to PingCAP.
 	EnableTelemetry bool `toml:"enable-telemetry" json:"enable-telemetry"`
+	// ServerIDTTL is the Time-To-Live of serverID in PD.
```

Comment:
Suggest to improve the comment so that readers that do not know the background or the design of this PR still knows the usage of this configuration.

Reply:
En...How to do this ?

Reply:
Just leave it as a const in go?